### DISCUSSION TITLE
Error when displaying tables - anyone else having this issue?

### DISCUSSION THREAD

**Discussion by benjamin-hauser** [posit-dev/great-tables#361] [Q&A] [answered]
**I get this error anytime im using polars with great tables.**

`TypeError: ColumnFactory.__new__() missing 1 required positional argument: 'name'`

Sample code below from site

```
import polars as pl
from great_tables import GT, md, html
from great_tables.data import islands

islands_mini = (
    pl.from_pandas(islands).sort("size", descending=True)
    .head(10)
)

(
    GT(islands_mini, rowname_col="name")
    .tab_header(
        title="Large Landmasses of the World",
        subtitle="The top ten largest are presented"
    )
    .tab_source_note(source_note="Source: The World Almanac and Book of Facts, 1975, page 406.")
    .tab_source_note(
        source_note=md("Reference: McNeil, D. R. (1977) *Interactive Data Analysis*. Wiley.")
    )
    .tab_stubhead(label="landmass")
    .fmt_integer(columns="size")
)
```

**Comment by jrycw** [comment, marked as answer]
@benjamin-hauser, could you try using `polars==0.20.26`? The issue might be due to the latest version of `Polars` updating the selection logic based on our request. We're working on a PR (#360 ) to provide a quick patch.

**Comment by machow** [comment]
Hmm.. I'm surprised by that specific error. I tried it with polars v0.20.30, and wasn't able to reproduce. I suspect upgrading polars should fix?

**Comment by jrycw** [comment]
I'll close the discussion since the issue appears to be resolved.
